src/client/src/mockData/mockVsCodeApi.ts

```diff
@@ -40,7 +40,7 @@ const mockVsCodeApi = () => ({
               command: EXTENSION_COMMANDS.GET_DEPENDENCY_INFO,
               payload: {
                 dependency: "python",
-                installed: false
+                installed: true
```

Comment:
Any specific reason to change the mock data?

Reply:
No specific reason, but on my computer I usually have node installed and no python so I can check that through the extension. On the browser, I can check the opposite (node not installed, and python installed warnings)